modules/gapi/include/opencv2/gapi/video.hpp

```diff
@@ -18,42 +18,83 @@
 namespace cv { namespace gapi {
 namespace  video
 {
-    using GOptFlowLKOutput = std::tuple<cv::GArray<cv::Point2f>,
-                                        cv::GArray<uchar>,
-                                        cv::GArray<float>>;
-
-    G_TYPED_KERNEL(GCalcOptFlowLK,
-                   <GOptFlowLKOutput(GMat,GMat,cv::GArray<cv::Point2f>,cv::GArray<cv::Point2f>,Size,
-                                     int,TermCriteria,int,double)>,
-                   "org.opencv.video.calcOpticalFlowPyrLK")
+using GBuildPyrOutput  = std::tuple<GArray<GMat>, GScalar>;
+
+using GOptFlowLKOutput = std::tuple<cv::GArray<cv::Point2f>,
+                                    cv::GArray<uchar>,
+                                    cv::GArray<float>>;
+
+G_TYPED_KERNEL(GBuildOptFlowPyramid, <GBuildPyrOutput(GMat,Size,GScalar,bool,int,int,bool)>,
```

Comment:
@dmatveev should I use GOpaque instead of GScalar to cover the `int` variable "maxLevel"? the same question in calcOpticalFlow
All the previous kernels use GScalar, but I think only because GOpaque didn't exist that time

Reply:
That's true, and `GOpaque` looks a better alternative (since it is typed).

However, `GOpaque` doesn't support value-initialization while the `GScalar` does. It may be more useful sometimes.

Let's continue this way and port to `GOpaque` later, after we introduce value initialization for it.